src/index.css

```diff
@@ -28,3 +28,44 @@
     filter: invert(1);
   }
 }
+
+.data-dictionary p {
+  margin: 0.5em 0;
+  padding: 0
+}
+
+.data-dictionary main h3 {
+  font-weight: bold;
+  font-size: 1.1em;
+  margin-top: 0.5em;
+}
+
+.data-dictionary cite, .data-dictionary .literal {
+  font-family: monospace;
+  font-style: normal;
+}
+
+.data-dictionary dl {
+  display: grid;
+  grid-gap: 0px 8px;
+  grid-template-columns: max-content;
+}
+
+.data-dictionary dt {
+  text-align: right;
+}
```

Comment:
nit: I think this looks a little better?
```suggestion
.data-dictionary dt {
  text-align: right;
  font-weight: bold;
}
```
Before:
<img width="1298" height="940" alt="image" src="https://github.com/user-attachments/assets/f13ed8db-33c3-4d16-bdd5-4977c6ba3206" />

After:
<img width="1298" height="940" alt="image" src="https://github.com/user-attachments/assets/4baf9db7-7ceb-49b6-bdf7-2e0953fda6a7" />